packages/app-project/src/screens/ClassifyPage/ClassifyPage.spec.js

```diff
@@ -10,6 +10,9 @@ import YourStats from './components/YourStats'
 import ConnectWithProject from '@shared/components/ConnectWithProject'
 import ProjectStatistics from '@shared/components/ProjectStatistics'
 
+// TODO: is there a better way to find the dynamically imported classifier?
+const ClassifierDisplayName = 'ForwardRef(LoadableComponent)'
```

Comment:
You can export the dynamically imported component, line 18 in the `ClassifyPage` file and then it can be imported here for the tests as `ClassifierWrapper`

Reply:
I like that a lot better than using the display name. When I’ve tried this before, enzyme doesn’t recognise the export as a component.

Reply:
That totally worked. I have no idea why enzyme was complaining before.